examples/reaction_prediction/rexgen_direct/README.md

```diff
@@ -149,6 +149,14 @@ New datasets should be processed such that each line corresponds to the SMILES f
 The reactants are placed before `>>` and the product is placed after `>>`. The reactants are separated by `.`. 
 In addition, atom mapping information is provided.
 
+**Notice**
+
+```
+The map ids in the rxn should be consecutive， or it will report [the molAtomMapNumber issue](https://github.com/awslabs/dgl-lifesci/issues/33).
+
+To avoid the problem, you could convert the raw rxn smiles with explicit hydrgogen atoms to the rxn smiles without hydrogen atoms by RDKit befor adding map ids for the rxn smiles.
```

Comment:
I don't think we have mentioned rxn smiles anywhere. If we want to use the term, we will want to define it first.

Reply:
use reaction smiles instead of rxn smiles?